dev/tasks/tasks.yml

```
@@ -1788,6 +1791,12 @@ tasks:
         UBUNTU: 18.04
       run: ubuntu-docs
 
+  ############################## vcpkg tests ##################################
+
+  test-build-vcpkg-win:
+    ci: github
+    template: vcpkg-tests/test-vcpkg-win.yml
```

We use `${CI_PLATFORM}.${PLATFORM}.yml` format for template file name:

```suggestion
    template: vcpkg-tests/github.windows.yml
```

---

Renamed in 4c4247